src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/Controllers/AuthenticationController.cs

```diff
@@ -130,18 +130,20 @@ public async Task<ActionResult> Get(string goTo)
         [HttpGet("refresh")]
         public async Task<ActionResult> RefreshJWTCookie()
         {
+            _logger.LogInformation($"Starting to refresh token...");
             ClaimsPrincipal principal = HttpContext.User;
 
             await HttpContext.SignInAsync(
                 JwtCookieDefaults.AuthenticationScheme,
                 principal,
                 new AuthenticationProperties
                 {
-                    ExpiresUtc = DateTime.UtcNow.AddMinutes(int.Parse(_generalSettings.GetJwtCookieValidityTime)),
+                    ExpiresUtc = DateTime.UtcNow.AddMinutes(30),
```

Comment:
IS there a reason we go away from config?

Reply:
used it for testing locally, will update it with config